packages/block-editor/src/components/rich-text/index.js

```diff
@@ -115,6 +115,14 @@ function RichTextWrapper(
 	},
 	forwardedRef
 ) {
+	if ( multiline ) {
+		deprecated( 'wp.blockEditor.RichText multiline prop', {
+			since: '14.1',
```

Comment:
Searching through the codebase, I'd say the convention is to use WP versions here. The only two instances of Gutenberg version numbers (12.6 and 12.7) are in `deprecated` calls with the accompanying `plugin: 'Gutenberg'` option.

Reply:
Oh. Adjusted it to use WP version numbers and added a version prop.